docs/usage.md

```diff
@@ -192,12 +192,6 @@ TREATMENT_REP3,AEG588A6_S6_L003_R1_001.fastq.gz,
 TREATMENT_REP3,AEG588A6_S6_L004_R1_001.fastq.gz,
 ```
 
-| Column         | Description                                                                                                                                                                            |
-|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `sample`       | Custom sample name. This entry will be identical for multiple sequencing libraries/runs from the same sample. Spaces in sample names are automatically converted to underscores (`_`). |
-| `fastq_1`      | Full path to FastQ file for Illumina short reads 1. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
-| `fastq_2`      | Full path to FastQ file for Illumina short reads 2. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
-
 An [example samplesheet](../assets/samplesheet.csv) has been provided with the pipeline.
```

Comment:
Does this example also contain a `nanoseq` example?

Reply:
Yes, [`../assets/samplesheet.csv`](https://github.com/nf-core/nanoseq/blob/dev/assets/samplesheet.csv) is an example samplesheet.csv specifically for `nanoseq`